controllers/vgmanager_daemonset.go

```diff
@@ -33,12 +33,12 @@ var (
 	DevDirVolName = "device-dir"
 	SysVolName    = "sys"
 
-	LVMdDir    = "/mnt/lvmd/"
+	LVMdDir    = "/etc/topolvm"
```

Comment:
if this is the directory where the lvmd config file is stored, then this can cause problems, because we already have another constant with the complete file path. So in case of change, everybody must know that we need to change this other constant.
I suggest to use only the "LvmdConfigFile" constant , and if it is needed extract the base path from the constant.

Reply:
Good point. I would like to take this up in a follow up PR.